pages/api/ticket/[uuid].ts

```diff
@@ -91,8 +107,11 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
     const jwToken = await createJWT({ ...ticket, ticket: html, qrCode })
     res.json({ ticketdata: jwToken })
   } catch (error) {
-    if (error.name === 'TypeError') {
-      return res.status(404).json({ error: 'invalid ticket UUID' })
+    if (error instanceof TicketNotFoundError) {
+      return res.status(404).json({ error: 'Ticket UUID incorrect' })
+    }
+    if (error instanceof TypeError) {
+      return res.status(500).json({ error: 'Internal server error' })
```

Comment:
I would probably response with error.message so its easier to see what caused the internal server error